pkg/kubelet/apis/kubeletconfig/validation/validation_others.go

```diff
@@ -0,0 +1,28 @@
+// +build !windows
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"k8s.io/kubernetes/pkg/kubelet/apis/kubeletconfig"
+)
+
+// ValidatePlatformConfiguration validates platform specific configuration and returns an error if it is invalid.
+func ValidatePlatformConfiguration(kc *kubeletconfig.KubeletConfiguration) error {
```

Comment:
We should probably put `KubeletConfiguration` somewhere in the name of this function, to keep with the naming scheme of `ValidateKubeletConfiguration`.

Since it's included in `ValidateKubeletConfiguration`, it can probably be private as well.

Reply:
We use `OS` instead of `Platform` in other contexts (see `osflags_windows.go`).

Reply:
ACK

Reply:
@mtaufen  Renamed to validateKubeletOSConfiguration. PTAL